chains/atomic/mocks_generate_test.go

```diff
@@ -0,0 +1,6 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
```

Comment:
Question on the copyright header:

1. Should we keep it for mocks_generate_test.go files?
2. Should we also add it to generated mocks or not really?

Reply:
We already don't have the header for the generated `pb.go` code - so I don't think we really need it on the generated mocks.

As far as the `mocks_generate_test.go` files, feels like we might as well have it.